05-setting-up.Rmd

```diff
@@ -70,8 +70,8 @@ You can take a look at this [template README](https://raw.githubusercontent.com/
 
 #### More about writing READMEs:
 
-- [How to write a good README file by Hillary Nyakundi](https://www.freecodecamp.org/news/how-to-write-a-good-readme-file/)
```

Comment:
@cansavvy ... idk why this got removed in this PR... I assume we want to keep it right?

Reply:
oh I see maybe this is a broken URL you fixed?

Reply:
looks like it isn't broken though: https://www.freecodecamp.org/news/how-to-write-a-good-readme-file/